tools/python.mk

```diff
@@ -74,3 +74,19 @@ install: $(VENV)
 .PHONY: .clean
 .clean:
 	rm -Rf $(VENV)
+
+DISTRO ?= bullseye
+DOCKER_RUN = docker run -it --rm \
+			--user $$(id -u):$$(id -g) \
+			--env HOME=/src/.docker/$(DISTRO) \
+			--volume $$(pwd)/..:/src \
+			--workdir /src/$(APP) \
+			ghcr.io/jooola/libretime-dev:$(DISTRO)
+
+docker-dev:
+	$(MAKE) clean
+	$(DOCKER_RUN) bash
+
+docker-test:
+	$(MAKE) clean
+	$(DOCKER_RUN) make test
```

Comment:
I've been using this for testing on other distros, not sure we should merge it though.

Reply:
Would it be helpful for the development of other features?

Reply:
Probably anytime we want to run tests on another distro, without spinning a full vagrant setup.

Reply:
We will always be able to remove it.